README.md

```diff
@@ -2,30 +2,24 @@
 
 ![Build](https://github.com/eganjs/synth-a-py/workflows/ci/badge.svg)
 
-Project configuration as code
+Daring to tackle the problem of complex project configuration through code!
 
-## Goals
+_synth-a-py_ is a framework to enable synthesising of project configuration using Python :snake:
 
-- [ ] Use synth-a-py to manage project configs
-  - Add support for:
-    - [x] LICENSE
-    - [x] TOML (for pyproject.toml)
-    - [x] YAML (for GitHub Actions config)
-      - [ ] GitHub Action workflow?
-    - [x] INI (for flake8/mypy config)
-    - [ ] Makefile
-    - [ ] .gitignore
-  - Add ./synth.py
-- Templates:
-  - [ ] Poetry
-  - [ ] setup.py
-  - [ ] Pipenv
-- In-repo examples:
-  - [ ] Minimal
-  - [ ] Monorepo
+The framework provides the means to define a project with multiple modules, that when synthesised, creates files such as `pyproject.toml`, `.github/workflows/{ci,cd}.yml` and `makefile`, all in the right places with consistency.
```

Comment:
```suggestion
The framework provides the means to define a project with multiple modules, that when synthesised, creates files such as `pyproject.toml`, `.github/workflows/{ci,cd}.yml` and `makefile`.
```

Reply:
I hope it's implied :)

Reply:
Given the choice I'd prefer to be explicit :stuck_out_tongue: